code/modules/mob/living/carbon/life.dm

```diff
@@ -239,7 +239,7 @@
 		if(prob(D.infectivity))
 			D.spread()
 
-		if(stat != DEAD)
+		if(stat != DEAD && !D.process_dead)

```

Comment:
shouldn't this be `(stat != DEAD || D.process_dead)`?

Reply:
Yes, it definitely should. /tg/ later changed it. https://github.com/tgstation/tgstation/pull/35618